jenkins-scripts/docker/gz_transport-compilation.bash

```diff
@@ -32,6 +32,10 @@ if [[ ${GZ_TRANSPORT_MAJOR_VERSION} -ge 6 ]]; then
   export NEED_C17_COMPILER=true
 fi
 
+if [[ ${GZ_TRANSPORT_MAJOR_VERSION} -ge 15 ]]; then
+  export ROS_DISTRO="jazzy"
```

Comment:
```suggestion
  export ROS_DISTRO_SETUP_NEEDED="jazzy"
```

I would not mess with defining ROS_DISTRO but seems safer to me to inject a new variable.

Reply:
Also a comment on why we need this ROS package

Reply:
renamed to `ROS_DISTRO_SETUP_NEEDED` and added comment. [e25781b](https://github.com/gazebo-tooling/release-tools/pull/1311/commits/e25781be102940ddb378492d09b179cbb9e595d0)